RTNetwork.cs

```diff
@@ -8,6 +8,34 @@
 
 namespace QTMRealTimeSDK.Network
 {
+    internal class Response

```

Comment:
This might as well be a struct (value type). It contains two small values and shouldn't really be shared through multiple references (it's most likely just used directly when evaluating response then discarded).

Reply:
Changed it to a struct